Plugin.pm

```diff
@@ -33,6 +33,8 @@ tie my %localizationTable, 'Tie::RegexpHash';
 	qr/^Livret Num.rique/i => 'PLUGIN_QOBUZ_BOOKLET'
 );
 
+$prefs->set('classicalGenres','') if (! $prefs->get('classicalGenres'));
```

Comment:
Just add a line to the below `init()` statement:

```perl
$prefs->init({
      classicalGenres => '',
      preferredFormat...
```

Reply:
Done.